### PR TITLE
MicroOS: create separate function for 10GB HDDs

### PR DESCRIPTION
**Pull request by jlausuch**
[10G](https://openqa.opensuse.org/tests/3237770)
[non 10G](https://openqa.opensuse.org/tests/3237771)

### CODE REVIEW

File: lib/main_micro_alp.pm [hunk 1]
```diff
@@ -66,55 +66,62 @@ sub load_boot_from_dvd_tests {
     }
 }
 
+sub load_10GB_installation_tests {
+    # boo#load_installation_tests
```
Comment (Vogtinator) [inline lib/main_micro_alp.pm:70]: Might as well fix the broken bugref - if you can find the right one?

File: lib/main_micro_alp.pm [hunk 2]
```diff
@@ -66,55 +66,62 @@ sub load_boot_from_dvd_tests {
...
+    loadtest 'installation/installation_overview';
+}
+
 sub load_installation_tests {
-    if (check_var('HDDSIZEGB', '10')) {
-        # boo#load_installation_tests
-        # undefined method "safe_copy" for nil:NilClass
-        # YaST2 crashes if disk is too small for a viable proposal
-        loadtest 'installation/welcome';
+    loadtest 'installation/welcome';
```
Comment (Vogtinator) [inline lib/main_micro_alp.pm:83]: Leftovers?

Comment (jlausuch) [reply lib/main_micro_alp.pm:83]: yes, sorry, amended.

File: lib/main_micro_alp.pm (hunk 1 above)
Comment (jlausuch) [reply lib/main_micro_alp.pm]: found it https://bugzilla.opensuse.org/show_bug.cgi?id=1099762

**Review by Vogtinator — APPROVED** [review]

**Review by mloviska — APPROVED** [review]

**Pull request merged**